### PR TITLE
[WEB-1302-41] Fix/address correction

### PR DESCRIPTION
**Pull request by pmdaly**
Updating addresses from Arbitrum to Ethereum for chainlink registry tests.

### CODE REVIEW

**Review by jstashh — APPROVED** [review]

File: tests/conftest.py [hunk 1]
```diff
@@ -21,16 +21,21 @@ def shared_setup(fn_isolation):
   pass
 
 
+@pytest.fixture
```
Comment (jstashh) [inline tests/conftest.py:24]: @pmdaly I thought this fix was already in master?

Comment (jstashh) [reply tests/conftest.py:24]: https://github.com/yearn/yearn-lens/pull/39/files

Comment (pmdaly) [reply tests/conftest.py:24]: It is already in master, I'm not sure why it's shown here as a change.

Comment (pmdaly) [reply tests/conftest.py:24]: It should have already been in master from a previous PR of mine.  I'm not sure why it's not in master currently.

Comment (pmdaly) [reply tests/conftest.py:24]: I'm not sure why it's showing up as a change here.  Possibly master moved prior to my PR? Maybe I need to rebase?

**Pull request merged**